pyanaconda/core/regexes.py

```diff
@@ -103,7 +103,7 @@
 # with a period, but it can end with one.
 # This regex uses negative lookahead and lookback assertions to enforce the
 # hyphen rules and make it way more confusing
-HOSTNAME_PATTERN_WITHOUT_ANCHORS = r'(?:(?!-)[A-Za-z0-9-]{1,63}(?<!-)(?:\.(?!-)[A-Za-z0-9-]{1,63}(?<!-))*\.?)'
+HOSTNAME_PATTERN_WITHOUT_ANCHORS = r'(?:(?!-)[A-Za-z0-9-]{1,63}(?<!-)(?:\.(?!-)[A-Za-z0-9-]{1,63}(?<!-))*)'
```

Comment:
Every time I'm seeing a change in these rexeps I am getting hives. :(

Don't get me wrong, I like regexps but these giant things are not good way to go.

Reply:
I think they are pretty ;-)

Reply:
With some assistive technology, this one is still quite readable...